src/cube.js

```diff
@@ -87,6 +87,18 @@ class Cube {
         this.storedMeasuresRules[measureId] = rules;
     }
 
+    copyToStoredMeasure(
```

Comment:
Now that we are here I was wondering if it would make sense to have a convertToStoredMeasure variant as well :thinking:

Reply:
yes